docs/bosh-env/README.md

```diff
@@ -9,21 +9,19 @@ The `bosh-env` command setup environment variables to target bosh director and/o
-  --target, -t, OM_TARGET                string  location of the Ops Manager VM
-  --trace, -tr                           bool    prints HTTP requests and response payloads
-  --username, -u, OM_USERNAME            string  admin username for the Ops Manager VM (not required for unauthenticated commands)
-  --version, -v                          bool    prints the om release version (default: false)
-
-Command Arguments:
-  --ssh-private-key, -i         string             location of ssh private key
-  --shell-type                  string             Prints for the given shell (posix|powershell)
+Usage: om [options] bosh-env [<args>]
+  --client-id, -c, OM_CLIENT_ID                          string  Client ID for the Ops Manager VM (not required for unauthenticated commands)
+  --client-secret, -s, OM_CLIENT_SECRET                  string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)
+  --connect-timeout, -o, OM_CONNECT_TIMEOUT              int     timeout in seconds to make TCP connections (default: 10)
+  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE  string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)
+  --env, -e                                              string  env file with login credentials
+  --help, -h                                             bool    prints this usage informa        string  Client ID for the Ops Manager VM(not required for unauthenticated commands)
+  --client-secret, -s, OM_CLIENT_SECRET        string  Client Secret for the Ops Manager VM (not required for unauthenticated commands)  --connect-timeout, -o, OM_CONNECT_TIMEOUT
+        int     timeout in seconds to make TCP connections (default: 10)  --decryption-passphrase, -d, OM_DECRYPTION_PASSPHRASE  string  Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands)  --env, -e        string  env file with login credentials
+  --help, -h        bool    prints this usage information (default: false)  --password, -p, OM_PASSWORD
+        string  admin password for the Ops Manager VM (not required for unauthenticated commands)  --request-timeout, -r, OM_REQUEST_TIMEOUT        int     timeout in seconds for HTTP requests to Ops Manager (default: 1800)  --skip-ssl-validation, -k, OM_SKIP_SSL_VALIDATION      bool    skip ssl certificate validation during http requests (default: false)
+  --target, -t, OM_TARGET        string  location of the Ops Manager VM  --trace, -tr, OM_TRACE        bool    prints HTTP requests and response payloads  --username, -u, OM_USERNAME
+        string  admin username for the Ops Manager VM (not required for unauthenticated commands)
+  --version, -v        bool    prints the om release version (default: false)Command Arguments:  --shell-type           string  Prints for the given shell (posix|powershell)  --ssh-private-key, -i  string  Location of ssh
```

Comment:
Should Command Arguments be on a new line?

Reply:
Yep, sorry – I was trying to get this in quickly, didn’t proofread. I’ll push a clean-up commit shortly